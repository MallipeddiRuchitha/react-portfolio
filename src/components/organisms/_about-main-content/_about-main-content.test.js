import React from 'react';
import { shallow ,mount} from 'enzyme';
import About from './_about-main-content';

describe("About component",() =>{
    test("Matches the snapshot",() => {
        const about = shallow(<About />);
        expect(about).toMatchSnapshot();

    })
    test("about element is rendered",() => {
        const about= shallow(<About />);
        expect(about.find("#aboutMainContent").exists()).toEqual(true);
        console.log(about.debug());
    });
   
   

});