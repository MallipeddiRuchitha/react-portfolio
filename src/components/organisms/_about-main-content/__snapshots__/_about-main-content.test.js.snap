// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component Matches the snapshot 1`] = `
<div
  className="makeStyles-aboutMainContent-1"
  id="aboutMainContent"
>
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-about-2"
    variant="subtitle2"
  >
     I am Ruchitha from Keshav Memorial Institute of Technology.I'm passionate about coding and learning new technologies.In free time I like to chill at home watching movies and colouring .I love to be surrounded by positive and motivating people. 
  </WithStyles(ForwardRef(Typography))>
  <div
    className="makeStyles-list-4"
  >
    <AboutCard
      array={
        Array [
          "Java",
          "DBMS",
          "React",
          "Spring",
          "Spring boot",
          "REST",
          "Networking",
          "Linux",
          "HTML,CSS",
        ]
      }
      heading="Technical Skills"
    />
    <AboutCard
      array={
        Array [
          "Detection and Prediction of Clouds",
          "Gate Pass Management System",
          "Making Data Based Decisions Using Google Analytics",
          "Online Doctor Appointment",
          "Spring boot REST API which gives information about movies and associated actors",
          "Spring boot application which gives information about courses and related topics",
        ]
      }
      heading="Projects"
    />
  </div>
</div>
`;
